blank-canvas-blocks/experimental-theme.json

```diff
@@ -129,6 +129,10 @@
 			},
 			"margin": {
 				"baseline": "10px",
+				"bodyHorizontal": "0px",
+				"bodyVertical": "0px",
+				"bodyHorizontalDesktop": "0px",
```

Comment:
I'm concerned that by adding more custom variables within the theme that are media query specific, we are going down a similar path to alignments that won't be compatible with FSE.

Reply:
Since we have a standard method of describing padding perhaps it would be better to define this just like any other element.  And to take a note from the button hover state refactor, perhaps could also express the mobile 'state' in the same way.  i.e.
```
		"custom": {
			"alignment": {
				"alignedMaxWidth": "50%"
			},
			"body": {
				"spacing": {
					"padding": {
						"top": "0px",
						"bottom": "0px",
						"left": "0px",
						"right": "0px"
					}
				},
				"mobile": {
					"spacing": {
						"padding": {
							"top": "0px",
							"bottom": "0px",
							"left": "0px",
							"right": "0px"
						}
					}
				}
			},
```